var/spack/repos/builtin/packages/recorder/package.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Recorder(AutotoolsPackage):
+    """A multi-level library for understanding I/O
+    activity in HPC applications."""
+
+    homepage = "https://github.com/uiuc-hpc/Recorder"
+    url      = "https://github.com/uiuc-hpc/Recorder/archive/v2.1.4.tar.gz"
+
+    maintainers = ['wangvsa']
+
+    version('2.1.4', sha256='f66756595a7f310929c247ae03fd08a18d9843f578fffa1e3072f557bf5a158e', preferred=True)
```

Comment:
No need for `preferred=True` here, if there's only a single version in the package it will always be used by default.

Reply:
Got it. I put it there so I won't forget this feature later when I have more versions.

Reply:
Spack automatically defaults to the latest stable release, but if for some reason you want an older release to be installed by default then you can use `preferred`.

Reply:
I have removed preferred.